src/main/java/mat/server/clause/QDSAttributesServiceImpl.java

```diff
@@ -7,13 +7,10 @@
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;

```

Comment:
star imports are not recommended. could you update your intellj settings?

Reply:
Sure